### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by Wangzeyu6** [Manaro-Alpha/DreamWaQ#4]
I tried to incorporate your calculation method of the loss function into my code, but I encountered some problems. Will a huge loss value suddenly occur during the training process? Have you ever encountered this problem?

![Image](https://github.com/user-attachments/assets/ce3d0f6a-0071-4da3-9c31-edc4adbd18da)

**Comment by hanlinniu** [comment]
Hi, I found even with the same policy and I didn't change anything, I got different reward figures, did you have the same problem?

<img width="348" height="275" alt="Image" src="https://github.com/user-attachments/assets/a03aafee-7a21-4856-aa0f-724260c9ff52" />

**Comment by Wangzeyu6** [comment]
> 您好，我发现即使使用相同的政策，而且我没有做任何更改，得到的奖励金额也不同，您遇到过同样的问题吗？
> 
> <img alt="图像" width="348" height="275" src="https://private-user-images.githubusercontent.com/62351309/468120052-a03aafee-7a21-4856-aa0f-724260c9ff52.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjQ5NDE5NTgsIm5iZiI6MTc2NDk0MTY1OCwicGF0aCI6Ii82MjM1MTMwOS80NjgxMjAwNTItYTAzYWFmZWUtN2EyMS00ODU2LWFhMGYtNzI0MjYwYzlmZjUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMjA1VDEzMzQxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRiOTNjMDAwNDYwYmVjZmI3ZmZhMTI3NWQ2ZjY2NDUyM2ExMTg1OTgzZDY0YjQyYzJlOGM1Y2FmYmM3ZTJjYTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ykK1jqC4jcQVrB5Kz2zc7RsLnMKhT3805DTkIe0ZXQ0">

你这个可能是由于随机数种子引起了，前提是你没有动其他代码，以及开启了随机种子
